controller/handler/patient-csv.js

```diff
@@ -107,8 +116,12 @@ function patientCSV (request, reply) {
             // send back updated data
             return questionOption;
         });
+        const property = ['id', 'questionText'];
+        const rowsm = deduplicate(allOptionsWithAnswers, property);
```

Comment:
change `const allOptionsWithAnswers` to `let allOptionsWithAnswers`
and assign deduplicate to `allOptionsWithAnswers`